### PR TITLE
commands: extract to pkg/

### PR DESCRIPTION
**Pull request by mgood**
Move `commands` out of chromestage into `pkg/` so that the common code can be
shared with bridge2.

Moved `Initialize` methods out of the `commands` package to simplify use when
they're not used directly as Tractor components.

### CODE REVIEW

File: images/chromestage/go.mod [hunk 1]
```diff
@@ -19,3 +20,5 @@ require (
 	golang.org/x/net v0.17.0 // indirect
 	golang.org/x/sys v0.13.0 // indirect
 )
+
+replace github.com/ajbouh/substrate/pkg/commands => ../../pkg/commands
```
Comment (mgood) [inline images/chromestage/go.mod:24]: @ajbouh I copied the pattern used for including `cueloader` here. This isn't needed when working locally with the `go.work` file, but was necessary for the Dockerfile build.

File: images/primers/docs/components/template.js [hunk 2]
```diff
@@ -415,8 +415,8 @@ export function defineDOMSurface({width, height, useChromestage}) {
             method: "POST",
             headers: {'Content-Type': "application/json"},
             body: JSON.stringify({
-              Command: command,
-              Parameters: parameters,
+              command,
+              parameters,
```
Comment (mgood) [inline images/primers/docs/components/template.js:419]: Is there something I should do to test this change? This was the only place I saw in here that was calling the Chromestage API.

File: images/chromestage/Dockerfile [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
+RUN go mod download
 
-# Get dependancies - will also be cached if we won't change mod/sum
+WORKDIR /go/src/github.com/ajbouh/substrate/images/chromestage/
+COPY images/chromestage/go.mod images/chromestage/go.sum ./
 RUN go mod download
 
-COPY images/chromestage/cmd cmd
-COPY images/chromestage/commands commands
-RUN CGO_ENABLED=0 go install -installsuffix "static" ./cmd/chromestage
```
Comment (ajbouh) [inline images/chromestage/Dockerfile:14]: Why did you remove `-installsuffix "static"`?

File: images/primers/docs/components/template.js (hunk 2 above)
Comment (ajbouh) [reply images/primers/docs/components/template.js:419]: Loading /primers/A2/ should test it, I think

**Review by ajbouh — COMMENTED** [review]
Change overall looks fine
